sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -128,15 +128,15 @@ class Pipeline():
   """
 
   # _default_pipeline is set when it (usually a compiler) runs "with Pipeline()"
-  _default_pipeline = None
+  _default_pipeline: 'Pipeline' = None
 
   @staticmethod
-  def get_default_pipeline():
+  def get_default_pipeline() -> 'Pipeline':
     """Get default pipeline. """
     return Pipeline._default_pipeline
 
   @staticmethod
-  def add_pipeline(name, description, func):
+  def add_pipeline(name: str, description: str, func) -> None:
```

Comment:
Nit: It's better to have refactorings and actual changes in separate PRs. Minimizing the PR diff is important for making the reviewing easier.

Reply:
Reverted.